instructor/process_response.py

```diff
@@ -193,6 +201,12 @@ def handle_response_model(
         if is_simple_type(response_model):
             response_model = ModelAdapter[response_model]
 
+        if is_typed_dict(response_model):
+            response_model: BaseModel = create_model(
```

Comment:
Maybe it would be easier to use RootSchema?
```
response_model = RootModel[response_model]
```

Reply:
But in this case, the value should taken from `response_model.root`, so we need to have another `if` 🤔

Reply:
Nevermind, it doesn't work